<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_city</class>
 <widget class="QWidget" name="add_city">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>237</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>237</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>237</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>237</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>772</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string>Add City</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_main">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>680</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(224, 204, 190);
    border-style: solid;
    color:  rgb(60, 54, 51);
    border-width: 5px;
    border-radius: 20px;
    border-color: rgb(116, 114, 100);
    font: 48px;
    padding: 6px;
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_name">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>438</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string>Enter City Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(224, 204, 190);
    border-style: solid;
    color:  rgb(60, 54, 51);
    border-width: 5px;
    border-radius: 20px;
    border-color: rgb(116, 114, 100);
    font: 48px;
    padding: 6px;
</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>290</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string>Connect Two Cities:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="city_1">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>378</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>370</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string>City 1 Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>442</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string>City 2 Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="city_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>450</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="distance">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>520</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>512</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string>Distance:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>580</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(224, 204, 190);
    border-style: solid;
    color:  rgb(60, 54, 51);
    border-width: 5px;
    border-radius: 20px;
    border-color: rgb(116, 114, 100);
    font: 48px;
    padding: 6px;
</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string>Add a New City:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="add_warning">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>590</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="connect_warning">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>660</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:  rgb(60, 54, 51);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
